modules/file-system/filestore/variables.tf

```diff
@@ -114,6 +114,16 @@ variable "connect_mode" {
   }
 }
 
+variable "nfs_export_options" {
+  description = "Define NFS export options."
+  type = object({
```

Comment:
`nfs_export_options` block can be repeated multiple times (up to 10), let's make it `list` of objects instead.

Reply:
Made it a non-nullable list.